typescript/looker/rtl/nodeTransport.ts

```diff
@@ -38,6 +38,21 @@ import rq, { Response, Request } from 'request'
 import rp from 'request-promise-native'
 import { PassThrough, Readable } from 'readable-stream'
 import { BaseTransport } from './baseTransport'
+import {ICryptoHash} from "./cryptoHash";
+import nodeCrypto from 'crypto';
```

Comment:
I wish IntelliJ would respect prettier settings and stop injecting trailing `;`

I need to fix `yarn prettier:write` from the base. It complains about an invalid version file.